packages/react-components/src/templates/__tests__/SharedResearchPageBody.test.tsx

```diff
@@ -29,10 +29,25 @@ const props: Omit<ComponentProps<typeof SharedResearchPageBody>, 'children'> = {
   renderPageHref: (index) => `#${index}`,
 };
 
-it('renders multiple library cards', () => {
-  const { queryAllByRole } = render(<SharedResearchPageBody {...props} />, {
-    wrapper: StaticRouter,
-  });
+it('renders multiple shared outputs cards in card view', () => {
```

Comment:
It looks like these do not actually check any difference so they would pass with `isListView` toggled the other way?